### PR TITLE
PROD-1062 feat(cookie consent): Add checkbox list

### PR DESCRIPTION
**Pull request by renanbronchart**

### CODE REVIEW

File: src/components/GoCookieConsent/GoCookieConsent.vue [hunk 1]
```diff
@@ -153,13 +246,56 @@ export default {
...
+    <div
+      v-for="(value, name, key) in model"
+      :key="key"
+    >
+      {{ name }}: {{ value }}
+    </div>
     <go-cookie-consent
       :textContent="text"
       :acceptLabel="labelAccept"
       :privacyData="{url: 'master.agorize.com', label: 'Our privacy content'}"
```
Comment (MatthieuGC) [inline src/components/GoCookieConsent/GoCookieConsent.vue]: We shouldn't have static urls

Comment (MatthieuGC) [inline src/components/GoCookieConsent/GoCookieConsent.vue]: We shouldn't have static urls

Comment (renanbronchart) [reply src/components/GoCookieConsent/GoCookieConsent.vue]: This is only for documentation @MatthieuGC

File: src/components/GoCookieConsent/GoCookieConsent.vue [hunk 2]
```diff
@@ -7,10 +7,37 @@
...
-          <p
-            class="text-white m-r-xs"
-            v-html="textContent"
-          />
+          <div>
+            <p
+              v-if="isVueComponent"
+              class="text-white m-r-xs"
+            >
+              <component :is="textContent" />
```
Comment (MatthiasHe) [inline src/components/GoCookieConsent/GoCookieConsent.vue]: `textContent` name for a potential component name is not really a good thing I think.

**Pull request merged**